### ISSUE TITLE
`\label` equation did not render by mathjax

### ISSUE THREAD

**Issue by vanabel** [benweet/stackedit#44]
When I try the following equation taken from [mathjax](http://docs.mathjax.org/en/latest/tex.html?highlight=equation%20number#automatic-equation-numbering):

```
In equation \eqref{eq:sample}, we find the value of an
interesting integral:

\begin{equation}
  \int_0^\infty \frac{x^3}{e^x-1}\,dx = \frac{\pi^4}{15}
  \label{eq:sample}
\end{equation}
```

the equation will be correctly rendered by mathjax at the first time, but after add some new content, the mathjax render did not work for the equation, if I remove the `\label` tag, it will work finely. 

I am quite sure this is a problem with mathjax render, not markup or stackedit, since the same problem exist in [StackExchange](http://math.stackexchange.com/questions/ask).

But could you please help me to find the key points of the problem, and feedback to mathjax development group?

**Comment by takasimiz** [comment]
As @cben mentioned, #180's 
>benweet commented on Nov 15, 2013
>> Actually, can you try that:
>>```
>>    userCustom.onAsyncPreview = function(callback) {
>>         MathJax.Hub.Queue(["resetEquationNumbers", MathJax.InputJax.TeX]);
>>         callback();
>>     };
>>```
solved the probrem in my case.

**Comment by benweet** [comment]
Closing as too old.
